contrib/babelfishpg_tsql/src/tablecmds.c

```diff
@@ -39,6 +39,7 @@
 
 const char* ATTOPTION_BBF_ORIGINAL_NAME = "bbf_original_name";
 const char* ATTOPTION_BBF_ORIGINAL_TABLE_NAME = "bbf_original_rel_name";
+const char* BBF_TABLE_CREATE_DATE = "bbf_rel_create_date";
```

Comment:
Generally, variable name for attoption is prefixed with ATTOPTION so can we also use the same convention here?